### ISSUE TITLE
Camera background turns black when it focuses the target.

### ISSUE THREAD

**Issue by JuanBMMontesinos** [codefluegel/vuforia4unreal#6]
When I start the app with the camera turned on, as soon as I point to the target the background turns black.
What could be happening?﻿
My cellphone is an Asus Zenphone 2

**Comment by geovie** [comment]
Hi, have you tried the branch at https://github.com/codefluegel/vuforia4unreal/tree/fix/i1
Your problem could be solved by this branch, see also issue #1

**Comment by JulianoBarreto** [comment]
Hello, you need to configure your Vuforia license, otherwise, the camera turns black.
-> Open the Project, go to Project Settings > Plugins > Vuforia and insert your Vuforia License Key
Good luck.

Update: Sorry, I didn't read right. ONLY the background turns black, right?

**Comment by JuanBMMontesinos** [comment]
Yes, Juliano Barreto, I did that configuration like you said before.
Then as I said, when the app starts, the camera works fine, however when I point the target the virtual object appears but the real background turns black keeping the virtual object on the target.
Now I downloaded the new link passed by geovie and I will try again.
As soon as possible I will give a feedback
Thanks

**Comment by azinonos** [comment]
Hello JuanBMMontesinos, I am having the same exact issue, have you managed to fix yours? If so, what was the solution?

**Comment by saphiresouldier** [comment]
Hello @azinonos, could you try if https://github.com/codefluegel/vuforia4unreal/tree/fix/i1 fixes the issue for you?

**Comment by azinonos** [comment]
Thanks for the reply @saphiresouldier. After deleting the camera actor and creating a new one it seems to work, I will keep it in mind however in case it does it again.
